sequencer/batchbuilder.go

```diff
@@ -17,6 +17,10 @@ import (
 	"github.com/jackc/pgx/v4"
 )
 
+var (
+	getTxsLimit = uint64(1)
```

Comment:
Is not this var being used as a constant in the code?

Reply:
Yep, it its. Will change it to const.